testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -5217,6 +5217,24 @@ QUnit.module('DateBox number and string value support', {
         }
     });
 
+    QUnit.test('First century year value should works correctly', function(assert) {
+        try {
+            $('#dateBox').dxDateBox({
+                value: '3/16/1964',
+                min: new Date(-50, 1, 1),
+                displayFormat: 'shortdate'
+            });
+
+            const $input = $('#dateBox').find(`.${TEXTEDITOR_INPUT_CLASS}`);
+
+            $($input.val('1/1/15')).trigger('change');
+
+            assert.deepEqual($('#dateBox').dxDateBox('option', 'text'), '1/1/15');
```

Comment:
I think, we can replace this assert with `assert.ok(true, 'no errors has been raised')` in `finally` block for green device tests

Reply:
i fix it by "pickerType: 'calendar'"